lib/resty/openssl/include/stack.lua

```diff
@@ -50,7 +50,7 @@ if OPENSSL_11_OR_LATER and not BORINGSSL then
   _M.OPENSSL_sk_free = C.OPENSSL_sk_free
   _M.OPENSSL_sk_deep_copy = C.OPENSSL_sk_deep_copy
 elseif OPENSSL_10 or BORINGSSL then
-  ffi.cdef [[
+  local cdef = string.gsub([[
```

Comment:
IMO it'd be more maintainable to separate boringssl declarations in its own branch, similarly to other places.

Reply:
@gszr  Good point, will fix shortly.